src/components/PersonalData/PersonalData.test.tsx

```diff
@@ -51,7 +51,11 @@ describe('Personal Data component', () => {
     })
 
     it('has no a11y violations', async () => {
-      expect(await axe(html.container)).toHaveNoViolations()
+      // Bug with NextJS Link + axe :(
```

Comment:
Unrelated to current work, but we were getting an `act` warning when running unit tests, so fixed this